docs/development/getting-started.rst

```diff
@@ -142,6 +149,15 @@ Web container is listening on port 80. It's accessible at
     docker container with ``docker-machine env``
 
 
+Stopping Warehouse and other services
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the terminal where `make serve` is running, use `Control-C` to
+gracefully stop the Warehouse application.
+
+Or, from another terminal, use ``docker-compose stop``.
```

Comment:
There is also the Makefile command `make stop` available, not sure which should be preferred here.

Reply:
`make stop`, because that filters for `warehouse` things to stop. Thanks. Fixing.